src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -875,11 +875,6 @@ public static int Sum<T>(this ImmutableArray<T> items, Func<T, int> selector)
             return dictionary;
         }
 
-        internal static Location FirstOrNone(this ImmutableArray<Location> items)
-        {
-            return items.IsEmpty ? Location.None : items[0];
-        }
```

Comment:
removed this as there was only compiler usage (in debug only code) and 2 usages in ide code (which i just updated).